src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/ApprovedOrderNoticeOfHearingService.java

```diff
@@ -179,7 +179,11 @@ private Map<String, Object> addValuesToPlaceHoldersMap(CaseDetails caseDetails,
         placeholdersMap.put("CCDCaseNumber", caseDetails.getId());
         placeholdersMap.put("DivorceCaseNumber", nullToEmpty(caseDetails.getData().get(DIVORCE_CASE_NUMBER)));
         placeholdersMap.put("ApplicantName", documentHelper.getApplicantFullName(caseDetails));
-        placeholdersMap.put("RespondentName", documentHelper.getRespondentFullNameContested(caseDetails));
+        if (caseDataService.isConsentedApplication(caseDetails)) {
```

Comment:
This logic can be simplified - there's a method in the CaseDataService called buildFullRespondentName which does the consented/contested check for you

Reply:
Thanks, updated